snap/snapcraft.yaml

```diff
@@ -48,8 +48,6 @@ parts:
       - -Dsoup2=true
     build-packages:
       - libpolkit-gobject-1-dev
-    prime:
-      - usr/lib/*/libsnapd-glib.so.*
```

Comment:
Do we need to stage more than just the lib here?

Reply:
You are right. Fixed.